hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/FileSystemBasedLockProvider.java

```diff
@@ -96,6 +96,7 @@ public void close() {
     synchronized (LOCK_FILE_NAME) {
       try {
         fs.delete(this.lockFile, true);
+        fs.close();
```

Comment:
Usually there is no need to close the filesystem because there are some reuse strategy on the HADOOP CLASSPATH, if we close the fs instance, it could induce exception for other invokers that are using this fs instance.

Reply:
close it first